README.md

```diff
@@ -46,6 +46,13 @@ This was rewritten version of [ghprb-plugin](https://wiki.jenkins-ci.org/display
   - [x] have one Global configuration
   - [x] reuse credentials lookup
 
+## Installation
+
+The only release now is in [Experimental Update Center](https://jenkins-ci.org/content/experimental-plugins-update-center/).
+Which is also the preferred way of installation. 
+
+If you don't want to change your Update Center, you can the download the `.hpi` from a mirror http://mirrors.jenkins-ci.org/plugins/github-pullrequest/ and upload it to Jenkins Plugin Manager.
```

Comment:
I'd say `then` is implicit when you want to "download something" and "upload it". But I'm not a native.